docs/04-reference/wingcli.md

```diff
@@ -133,6 +133,25 @@ JavaScript bundles that include inflight code that executes on compute platform
 
 You can deploy your stack to GCP using Terraform ([example](/getting-started/aws)).
 
+### `awscdk` Target
+
+Compiles your program for AWS-CDK and run on AWS.
+
+Usage:
+
+```sh
+$ export CDK_STACK_NAME="my-project"
+$ wing compile --target awscdk cdk-app.w
+```
+
+The output includes both a AWS-CDK configuration file (under `target/<file name>.awscdk`) and
+JavaScript bundles that include inflight code that executes on compute platform such as AWS Lambda.
+
+You can deploy your stack to AWS using `cdk deploy` command
+```sh
+$ cdk deploy --app target/cdk-app.awscdk
+```
```

Comment:
Calling it `cdk-app.w` feels a bit redundant here 😛 

````suggestion
```sh
$ export CDK_STACK_NAME="my-project"
$ wing compile --target awscdk app.w
```

The output includes both a AWS-CDK configuration file (under `target/<file name>.awscdk`) and
JavaScript bundles that include inflight code that executes on compute platform such as AWS Lambda.

You can deploy your stack to AWS using `cdk deploy` command
```sh
$ cdk deploy --app target/app.awscdk
```
````